### ISSUE TITLE
OSR 4.50.4 - problèmes d'affichage

### ISSUE THREAD

**Issue by itsakifa** [opengisch/signalo#156]
4.50.4 : le roller n’est pas complet et le texte n’est pas dynamique
![image](https://github.com/opengisch/signalo/assets/4679915/479fd461-a4d0-4203-91c4-5a8568aacd60)

**Comment by itsakifa** [comment]
semble déjà être corrigé

**Comment by spch-GL** [comment]
Oui, effectivement, mais le blanc du cadre est maintenant transparent dans les 2 directions...
Copie d'écran de la version L après ouverture dans chromium: 

![Image](https://github.com/user-attachments/assets/37fc6742-411a-4925-86de-45f4d7c3bf71)
